src/app/pre-sale/components/pre-sale-widget/index.tsx

```diff
@@ -1,8 +1,14 @@
 'use client'
 
+import { BigNumber } from 'ethers'
```

Comment:
can you use bigint instead?

Reply:
Not yet because `transactionReview` placeholder is based on our quotes still using BigNumber. This import should be gone with the next iteration tho when returning the real transactionReview object.